### PR TITLE
feat: p2p banner section

### PR DESCRIPTION
**Pull request by hasan-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/pages/home/p2p-banner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                        </Typography.Heading>
+                        <Typography.Paragraph size="large">
+                            <Localize translate_text="_t_Connect with fellow traders and transfer money in minutes._t_" />
+                        </Typography.Paragraph>
+                        <Link url={{ type: 'internal', to: '/' }} size="large" textcolor="brand">
+                            <Localize translate_text="_t_Learn more >_t_" />
+                        </Link>
+                    </FlexBox>
+                    <Box
+                        className={dclsx('typography-align-center', p2p_qr_container)}
```
Comment (mohsen-deriv) [inline src/features/pages/home/p2p-banner/index.tsx:59]: please don't use `typography-align-center` here, pass it as props to typography. thanks :pray:

File: src/features/pages/home/p2p-banner/index.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                    align="center"
+                    gap="12x"
+                    padding_inline="8x"
+                    padding_block="40x"
+                    md={{ mr: '10x', padding: '0x' }}
+                    lg={{ mr: '40x' }}
+                >
+                    <FlexBox className={p2p_box_container} gap="8x" direction="col">
+                        <StaticImage
+                            src="../../../../images/common/home/deriv_p2p.png"
```
Comment (mohsen-deriv) [inline src/features/pages/home/p2p-banner/index.tsx:40]: use should handle the RTL image as well, if needed

File: src/features/pages/home/index.tsx [hunk 3]
```diff
@@ -33,6 +34,7 @@ const HomePage = () => {
             <TradeTypes />
             <OurPlatforms />
             <WhatOurClientsSay />
+            <P2PBanner />
```
Comment (mohsen-deriv) [inline src/features/pages/home/index.tsx:37]:
```suggestion
      {is_p2p_allowed_country && <P2PBanner />}
```
please remove the old one from here as well, thanks :pray:

File: src/features/pages/home/p2p-banner/index.tsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    return (
+        <Container.Fixed as="section" bgcolor="tertiary">
+            <FlexBox
+                container="fixed"
+                direction="col-reverse"
+                md={{ direction: 'row', justify: 'between' }}
+            >
+                <Box mt="9x" md={{ mt: '0x' }}>
+                    <StaticImage
+                        src="../../../../images/common/home/p2p_home_banner.png"
```
Comment (mohsen-deriv) [inline src/features/pages/home/p2p-banner/index.tsx:22]: you should handle the RTL image here as well, if needed. thanks :pray:

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]
please break down you components to separate files as well, it's hard to manage this way, more modules will help us handle the features better. thanks :pray:

File: src/features/pages/home/p2p-banner/index.tsx (hunk 2 above)
Comment (hasan-deriv) [reply src/features/pages/home/p2p-banner/index.tsx:40]: How can i do that?

Comment (mohsen-deriv) [reply src/features/pages/home/p2p-banner/index.tsx]: I think there is another image for RTL, you can use that.

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**